less/home-page.less

```diff
@@ -1,22 +1,62 @@
 .home {
 
+  header {
+    width: 1000px;
+    position: relative;
+    right: 75px;
+    
+    @media @mobile {
+      width: 100%;
+      right: 0;
+    }
+  }
+  .mainbus {
+    margin: 0 10%;
```

Comment:
You might find that if you do percentage based margins and paddings, you might find some unexpected behavior in how your elements are getting formatted as you shrink and expand the page.  I experienced this while looking at some other projects.